### ISSUE TITLE
Research our onBlur validation with keyboard navigation and how it works with screen readers

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#238]
### Description

We have gotten feedback that our validation messages can feel premature in some scenarios. This is mostly in cases where you use a keyboard to go trough the form. We should do some user research if there are anything we need to change about how we solve this today. 

There are a lot of similar solutions that might have done research on this already. We should reach out to them, test and check how they have solved this issue. 

### In scope

- [ ] Research and reach out to other similar solution
  - [ ] When are validation triggered? Is it the same on all components and form types (large and small pages)
  - [ ] How is it presented for screen readers?
  - [ ] If relevant run user test with expert users or plan new user test after changes
- [ ] Are there anything we need to change about our validations?
  - [ ] Create issues with specification
  

### Out of scope

This is a research issue, it should not handle the potensial fix

**Comment by olemartinorg** [comment]
Seems very related to the previous issue, #237 (somewhat duplicate, but this issue is broader). After #98, required error messages is no longer displayed when simply tabbing through the form, and since we don't send any changes to the server when there are no changes to send, simply tabbing through the form without changing anything will in most cases never show you any validation errors. Maybe an exception to this is JsonSchema-based validations? I haven't checked thorougly.
